.dockerignore

```diff
@@ -1,9 +0,0 @@
-venv/*
-venv2.7/*
-.DS_Store
-frontend/node_modules
-frontend/build
-package-lock.json
-./engine/extensions
-.env
-.env-hobby
```

Comment:
this seemed fairly outdated. In any case, the specified `context` for all of our published images is `engine/` (see pipelines in `.drone.yml`), thereby I don't think this file would be picked up